check-osp/check_osp_main.py

```diff
@@ -122,6 +133,15 @@ def main():
         if args.services:
             services_dict = json.loads(args.services.replace("'", "\""))
             measure_func(system, logger=logger, services=services_dict)
+        elif args.services_file:
+            try:
+                with open(args.services_file, "r") as stream:
+                    services_dict = yaml.safe_load(stream)
+                    measure_func(system, logger=logger, services=services_dict)
+            except IOError:
+                msg = "File {} does not exists ".format(args.services_file)
```

Comment:
Small typo: `msg = "File {} does not exist".format(args.services_file)`

Reply:
Grammar Nazi :1st_place_medal:  :tada: